### PR TITLE
fix: Trynow btn fix

### LINKED ISSUE #1012: [BUG][FormTribe 🔥][50 Points] Try it now button does not look right and text is very light
### Issue Summary

I want to fix this button @jobenjada 

image

As you can see the text is very light

### Steps to Reproduce

1. (for example) Went to ...
2. Clicked on...
3. ...
-

### Expected behavior

-

### Other information

_No response_

### Screenshots

_No response_

### Environment

- [X] Formbricks Cloud (app.formbricks.com)
- [X] Self-hosted Formbricks

### Desktop (please complete the following information)

```markdown
- OS:
- Node:
- npm:
```

### PR DESCRIPTION
**Pull request by ratishjain12**
## What does this PR do?

I have fixed this issue @jobenjada 

Fixes #1012 

image

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Chore (refactoring code, technical debt, workflow improvements)
- [x] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change adds a new database migration
- [ ] This change requires a documentation update

## How should this be tested?

- Test A
- Test B

### Required

- [ ] Filled out the "How to test" section in this PR
- [ ] Read [How we Code at Formbricks](<[https://github.com/formbricks/formbricks/blob/main/CONTRIBUTING.md](https://formbricks.com/docs/contributing/how-we-code)>)
- [ ] Self-reviewed my own code
- [ ] Commented on my code in hard-to-understand bits
- [ ] Ran `pnpm build`
- [ ] Checked for warnings, there are none
- [ ] Removed all `console.logs`
- [ ] Merged the latest changes from main onto my branch with `git pull origin main`
- [ ] My changes don't cause any responsiveness issues

### Appreciated

- [ ] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the Formbricks Docs if changes were necessary

### CODE REVIEW

File: apps/formbricks-com/components/shared/UseCaseCTA.tsx [hunk 1]
```diff
@@ -13,9 +13,9 @@ export default function UseCaseHeader({ href }: UseCaseCTAProps) {
       <Button variant="secondary" href={href}>
         Step-by-step manual
       </Button>
-      <div className="space-y-1 text-center">
+      <div className="space-y-1 text-center ">
         <Button
-          variant="darkCTA"
+          className="bg-gray-800 text-gray-500 hover:text-white"
```
Comment (ShubhamPalriwala) [inline apps/formbricks-com/components/shared/UseCaseCTA.tsx:18]: Can you make the text more lighter for the dark mode? It looks really off currently. And can you also handle the text coloring being dynamic as per the light/dark mode as well? The text isn't really visible in the light mode:
image

File: apps/formbricks-com/components/shared/Header.tsx
Comment (ShubhamPalriwala) [inline apps/formbricks-com/components/shared/Header.tsx:1]: Can you revert the changes that aren't needed from this file? I don't think we need them in the context of this PR and are not adding value to the affected UseCTA component as well.

**Review by ShubhamPalriwala — CHANGES_REQUESTED** [review]
Hey, thanks a lot for finding the bug and opening a PR as well. The button looks better now. A couple of small nits from my end, once that's done, we're good to go 🚀

**Comment by ratishjain12** [comment]
> Hey, thanks a lot for finding the bug and opening a PR as well. The button looks better now. A couple of small nits from my end, once that's done, we're good to go 🚀 

Sure 👍

**Comment by ratishjain12** [comment]
@ShubhamPalriwala  is this fine?
image

**Comment by ShubhamPalriwala** [comment]
Great work @ratishjain12, looks good now 🚀

**Review by ShubhamPalriwala — APPROVED** [review]

**Comment by ratishjain12** [comment]
> Great work @ratishjain12, looks good now 🚀

Thanks looking forward to contribute more

**Comment by ratishjain12** [comment]
> > Great work @ratishjain12, looks good now 🚀
> 
> Thanks looking forward to contribute more

@ShubhamPalriwala can you add hacktoberfest labels on this?

**Pull request merged**